dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -133,10 +146,298 @@ xref:../admin_guide/high_availability.adoc#admin-guide-high-availability[VIP] ad
+
+----
+$ oc expose dc postgresql --type=LoadBalancer --name=postgresql-ingress
+----
+
+Once the service is created, the external IP address is automatically allocated
+by the cluster and can be confirmed by running:
+
+----
+$ oc get svc postgresql-ingress
+----
+
+.Example oc get Output
+----
+NAME         CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+postgresql-ingress    172.30.74.106   10.9.54.100,10.9.54.100    5432/TCP    30s
+----
+
+Specifying the type `LoadBalancer` also configures the service with a
+`nodePort` value, allowing direct access to the service on each of the nodes the
+application is deployed on. To discover the node port automatically assigned,
+run:
+
+----
+$ oc export svc postgresql-ingress
+----
+
+.Example oc export Output
+----
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    app: postgresql-persistent
+    template: postgresql-persistent-template
+  name: postgresql-ingress
+spec:
+  deprecatedPublicIPs:
+  - 10.9.54.100
+  externalIPs:
+  - 10.9.54.100
+  ports:
+  - nodePort: 32439 <1>
+    port: 5432
+    protocol: TCP
+    targetPort: 5432
+  selector:
+    name: postgresql
+  sessionAffinity: None
+  type: LoadBalancer
+----
+<1> Automatically assigned port.
+
+A PostgreSQL client can now be configured to connect directly to any node using
+the value of the assigned `nodePort`.
+
+[[configuring-the-service-to-be-highly-available]]
+==== Configuring the Service to be Highly Available
+
+Instead of connecting directly to individual nodes, you can use one of
+{product-title}'s
+xref:../admin_guide/high_availability.adoc#admin-guide-high-availability[high
+availability] strategies by deploying the IP failover router to provide access
+services configured with external IP addresses. This allows cluster
+administrators the flexibility of defining the ingress points within a cluster,
+and ensuring the service is readily available.
+
+[NOTE]
+====
+Nodes that have IP failover routers deployed to them must be in the same *Layer
+2* switching domain for ARP broadcasts to communicate to switches what
+appropriate port the destination should flow to.
+====
+
+[CAUTION]
+====
+Only 254 addresses can be allocated in a single VLAN due to the restrictions
+placed by the Virtual Router Redundancy Protocol (VRRP), which is used by the
+IP failover router.
+====
 
-Use the same nodes as the router, but advertise them with VRRP and use
-the DNS address of the router nodes.
+Since routers require privileged access to the host, they are deployed to
+facilitate port binding. A separate service account called *ipfailover* will be
+used to run the pod containing the router.
+
+. Pre-configure the service account to be a member of the privileged security
+context constraint by executing the following command:
++
+----
+$ oadm policy add-scc-to-user privileged system:serviceaccount:default:ipfailover
+----
+
+. *Multicast* must be enabled on each node the router will be deployed to in order
```

Comment:
@knobunc VRP requires 244.0.0.18/32 which is multicast. The NICs must be configured for multicast (linux layer, not openshift)